src/main/java/io/codearte/jfairy/producer/company/CompanyProvider.java

```diff
@@ -67,8 +68,27 @@ private void generateDomain() {
 		if (domain != null) {
 			return;
 		}
-		domain = TextUtils.stripAccents(StringUtils.strip(StringUtils.deleteWhitespace(name.toLowerCase()), ".").replace("/", ""))
-				+ "." + dataMaster.getRandomValue(DOMAIN);
+//		domain = TextUtils.stripAccents(StringUtils.strip(StringUtils.deleteWhitespace(name.toLowerCase()), ".").replace("/", ""))
```

Comment:
It works only with Java 8. However, 2 years ago we started a discussion (#23) to drop support for Java 7 and it seems to be a good argument to do that.